server/datastore/mysql/migrations/tables/20220330100659_AddJobsTable.go

```diff
@@ -0,0 +1,35 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220330100659, Down_20220330100659)
+}
+
+func Up_20220330100659(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE jobs (
+    id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    name VARCHAR(255) NOT NULL,
```

Comment:
So if I get this correctly, and if we use the queue comparison, `name` here is like the name of a queue, so `jira` integrations would use that name to send jobs to process with the `jira` worker, right? So there would be (potentially) many rows in `jobs` with the `jira` name, we don't use deduplication at the moment so each call to `worker.QueueJob` would always create a new row here (I think that's fine, just thinking out loud and see if I understand the intent correctly). We might want to store the unique ID of the instance that processed the job as additional metadata (the random text called `ourIdentifier` that gets passed to `cronWorker`) to help troubleshooting, so we can easily filter on relevant logs for that particular job.

Reply:
Yes, there would be many jobs with the same `name`. I wouldn't refer to it as the queue though, since the entire jobs table is the queue that workers query for work. 

Instead of logging `ourIdentifier`, why don't we log the `jobID`? `ourIdentifier` gets randomly generated on startup. Also, it seems like the more obvious choice if you want to find out why a particular job failed ie if a job failed, look for the logs with that job id.

Reply:
Yeah indeed, logging something like `"job_id":<id>` would be enough for that.